programs/zbufftest.c

```diff
@@ -512,8 +514,8 @@ int main(int argc, const char** argv)
     int result=0;
     U32 mainPause = 0;
     const char* programName = argv[0];
-    ZSTD_customMem customMem = { ZBUFF_allocFunction, ZBUFF_freeFunction };
-    ZSTD_customMem customNULL = { NULL, NULL };
+    ZSTD_customMem customMem = { ZBUFF_allocFunction, ZBUFF_freeFunction, (void*)777 };
```

Comment:
I believe `NULL` is a more common value for such usage.

"Better debugging" is fine, it must just be temporary. The shipping code would be better with a more standard `NULL` value.